script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt

```diff
@@ -192,13 +195,15 @@ However...
 [sha:151b48044deb776835cddca22ad3679e13c97441]{
 -- Page 589, Offset 18048.
 -- 「ソレを生み出したものがいる……いや、死徒より上の立場の吸血鬼がいる、ってコトか」
+// regular added here for clarity, and to connect with the line a few above
 "But something originally sired them...
- So that means there must exist vampires that stand above the Dead Apostles, right?"
+ So that means there must exist vampires that stand above regular Dead Apostles, right?"
 }
 [sha:b8d10bfdc2ce71ef517a559f409ac4fea8906b99]{
 -- Page 589, Offset 18049.
 -- 「そうよ。それがⅨ階梯、祖。
-"That's right. You're talking about Rank IX, %{i}Ancestors%{/i}.
+// 'You're talking about' gives a weird impression.
+"That's right. These are Rank IX, %{i}Ancestors%{/i}.
```

Comment:
Reads fine to me, but then I probably wrote it

Reply:
I'm fine either way but "these" doesn't work